ssl-impl/src/test/java/io/crate/protocols/postgres/SslReqHandlerTest.java

```diff
@@ -120,8 +114,10 @@ public void testClassLoadingFallback() {
         }
     }
 
-    @Test(expected = SslConfigurationException.class)
+    @Test
     public void testClassLoadingWithInvalidConfiguration() {
+        expectedException.expect(SslConfigurationException.class);
+        expectedException.expectMessage("Failed to build SSL configuration");
```

Comment:
Yes, but I don't need to check the message at all. Just the exception type. It will break the test if we change that message...